docs/recipes.rst

```diff
@@ -216,6 +216,38 @@ The preceeding example makes `all` fields from ``CreateTodoSchema`` optional in
 			super(UpdateTodoSchema, self).__init__(partial=partial_arg, **super_kwargs)
 
 
+Structuring Larger Projects
+===========================
+
+When structuring larger projects, there may be more than one directory which has modules which will be loaded into the ``Registry``.  They still need to be imported, but if there is nothing else to do with them, they aren't used. Instead Flask-Rebar can import those modules.
+
+Let's take a look at this setup:
+
+.. code-block::
+
+	application/
+	├── app.py
+	├── config.py
+	├── database.py
+	├── rebar.py
+	├── handlers
+	│   └── create.py
+	├── models
+	│   └── data.py
+	└── schemas
+		├── create.py
+		└── sync.py
+
+Inside of rebar.py we can find this:
+
+.. code-block:: python
+
+	from flask_rebar import Rebar
 
+	rebar = Rebar()
+	registry = rebar.create_handler_registry(handlers='application.handlers')
```

Comment:
Can `handlers` just be passed in at the same time that `load_handlers` is called?

Reply:
Not in the current implementation, it is currently an attribute that lives in the registry, but there isn't a reason that it couldn't be changed.

My plan was to eventually use the handlers attribute for finding blueprint like objects, so my current preference is to implement it like this so that I don't have to make a backwards incompatible change in the future.

Reply:
Not sure what Barak is thinking exactly, but in this example, I'd also like the handler module to be passed directly to `load_handlers`. Though I see why you want the attribute on the registry itself.

Potentially related: is there a reason to have an explicit function to call for this rather than having the modules imported implicitly at some point during `init_app`?

Reply:
init_app would work for me as well.  But i think I would still like it to be an attribute on the registry, so that there is a saved record of what is going to be loaded at init time.

Reply:
Yeah, makes sense to me. I brought up implicitly loading the modules instead, because that might be a way to keep the handlers attribute on the registry while eliminating the extra explicit call to `load_handlers` which might not read as well without referencing the handlers. That's what I'm thinking anyway.

Reply:
:+1: yeah sounds good.